lib/src/test/java/com/dynatrace/metric/util/MetricValuesTest.java

```diff
@@ -26,10 +26,15 @@ public void testFormatDouble() {
     assertEquals("0", MetricValues.formatDouble(.000000000000000));
     assertEquals("1", MetricValues.formatDouble(1));
     assertEquals("1", MetricValues.formatDouble(1.00000000000000));
-    assertEquals("1.234567", MetricValues.formatDouble(1.234567));
-    assertEquals("1.234568", MetricValues.formatDouble(1.234567890));
-    assertEquals("-1.234567", MetricValues.formatDouble(-1.234567));
-    assertEquals("-1.234568", MetricValues.formatDouble(-1.234567890));
+    assertEquals("1.23456789", MetricValues.formatDouble(1.23456789000));
+    assertEquals("1.1234567890123457", MetricValues.formatDouble(1.1234567890123456789));
+    assertEquals("-1.23456789", MetricValues.formatDouble(-1.23456789000));
+    assertEquals("-1.1234567890123457", MetricValues.formatDouble(-1.1234567890123456789));
+    assertEquals("200", MetricValues.formatDouble(200));
+    assertEquals("200", MetricValues.formatDouble(200.00000000));
+    assertEquals("1.0E12", MetricValues.formatDouble(1_000_000_000_000.));
+    assertEquals("1.234567E12", MetricValues.formatDouble(1_234_567_000_000.));
```

Comment:
```suggestion
    assertEquals("1.0E12", MetricValues.formatDouble(1_000_000_000_000d));
    assertEquals("1.234567E12", MetricValues.formatDouble(1_234_567_000_000d));
```
Otherwise it might look like a typo rather than an explicit double notation.